### PR TITLE
supermatter cascade - the end of the world

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
the supermatter cascade is a reality-ending catastrophe. Admin spawned only (events)
(idea is from /vg/)

## How This Contributes To The Skyrat Roleplay Experience
doomsday events should happen...

## Changelog

:cl:
add: added admin-spawned supermatter cascade
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Look
Not even critiquing the idea with this post; I love the concept, I have fond memories of its occasional use on oldbase.

But simply put, this just doesn't belong on a server aiming wherever the hell we're trying to aim, even if its admin-only trigger.
Events with this will devolve into nrp struggles to escape with ""pockets of roleplay"" that exist for minutes before being quite literally washed away. The strongest roleplay will be the leadup, which if they fail to prevent whatever from happening, could be much simpler than the sm sea's slow deathcrawl to end the round.

And from past experience, this will be used as soon as its implemented, leading to immediate complaints, leading to it being shunned lest the playerbase speak out against it again, leading to it seeing no use and this code being neglected in the end regardless.

Granted, you can take this with as much salt as you choose. I'm one of the less action-oriented here, its 5am, blahblah.
Just, do consider how the average player will see 'round ending admintrigger #xxx'
And hey, maybe I'm wrong. Idk its 5am

File: modular_skyrat/modules/apocalypse_supermatter_cascade/code/cascade.dm [hunk 1]
```diff
@@ -0,0 +1,181 @@
+/turf/open/indestructible/supermatter_cascade
+	name = "supermatter sea"
+	desc = "The end is night... this is a product from the supermatter slowly expanding to consume the whole universe."
```
Comment (SpookyTheFox) [inline modular_skyrat/modules/apocalypse_supermatter_cascade/code/cascade.dm]: "nigh", not "night". Minor spelling.

**Comment by SpookyTheFox** [comment]
On a technical note this should have a round end trigger of some kind after having run for a few minutes.
In my local testing, this seems to be causing an undesired interaction with ghost poly, spawning multiple of them and immediately spamming the engineering channel, since ghost of poly has a headset.

The actual tiles spawned are annoyingly flashy as well.

As a practical matter, you don't need to bother fixing it, because this should not be merged. Running this, even once, will greatly inflame player animosity towards events and staff in general, and is likely to result in events being put on hold or canceled _again_.

**Comment by LemonInTheDark** [comment]
Isn't this from vg?

**Comment by jjpark-kb** [comment]
> Isn't this from vg?

in a way yes. The idea is from /vg/ (which I should credit in the PR itself, but I put it in the readme).

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**